src/core/Subscription.ts

```diff
@@ -41,10 +41,19 @@ export class Subscription {
    * Get all subscriptions attached to a service account
    * @param requestBuilder Service account request builder
    */
-  public static async get(requestBuilder: RequestBuilder): Promise<Subscription[]> {
+  public static async get(requestBuilder: RequestBuilder, events?: EventName[]): Promise<Subscription[]> {
+    let params = {};
+    if (Array.isArray(events) && events.length) {
```

Comment:
If events is an array, then `length` is obviously defined

```suggestion
    if (Array.isArray(events) && events.length > 0) {
```

Reply:
Yes it is just a shortcut to check that the length is > 0

Reply:
Yes that's true!